R/dataProcessPlots.R

```diff
@@ -405,10 +507,14 @@ dataProcessPlots = function(
                             legend.size, label.color, cumGroupAxis, groupName,
                             lineNameAxis, yaxis.name)
       print(qc_plot)
+      plots <- c(plots, list(qc_plot))
       setTxtProgressBar(pb, i)
     } 
     close(pb)
   } 
+  if (isPlotly) {
+      return(plots)
```

Comment:
return is not necessary, we don't use it

Reply:
Commit [remove return statements](https://github.com/Vitek-Lab/MSstats/pull/107/commits/f9695cb099b2ff0f094132d00932ced607fd91bd)